apps/web/src/components/utils/SuccessOrError.tsx

```diff
@@ -0,0 +1,19 @@
+import { showNotification } from '@mantine/notifications';
+import { Check } from '../../design-system/icons/general/Check';
+import { ErrorIcon } from '../../design-system/icons';
+
+export function show(state: 'success' | 'error', message: string) {
```

Comment:
Maybe we should use a more descriptive name? Like showNotification or something else you can think of  that says what this function do :) 

Also I would give a different name to the file. Something that says its about notifications